keyboards/mehkee96/info.json

```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "",
```

Comment:
Ideally, this should be `"keyboard_name": "mehkee96",` and the line with the `bootloader` attribute deleted (it's deprecated and no longer used).

Totally optional though and won't affect functionality. I just forgot to mention these when we were talking on Discord.